narwhals/expr.py

```diff
@@ -1352,27 +1353,29 @@ def fill_null(
             ...     {
             ...         "a": [2, None, None, 3],
             ...         "b": [2.0, float("nan"), float("nan"), 3.0],
+            ...         "c": [1, 2, 3, 4],
             ...     }
             ... )
             >>> df = nw.from_native(df_native)
             >>> df.with_columns(
-            ...     nw.col("a", "b").fill_null(0).name.suffix("_nulls_filled")
```

Comment:
Sorry if stripping the "_nulls" can be misleading. I tried to keep the table at a reasonable width.